app/views/subscription_mailer/send_digest.html.erb

```diff
@@ -1,46 +1,38 @@
 <header style="text-align: center;color:black">
-	<img src="https://avatars2.githubusercontent.com/u/4621650?s=200&v=4" style="width: 100px;height: 100px">
-	<p style="font-size: 20px">[Public Lab] Your Weekly Digest</p>
+  <img src="https://avatars2.githubusercontent.com/u/4621650?s=200&v=4" style="width: 100px;height: 100px">
+  <p style="font-size: 20px">[Public Lab] Your Weekly Digest</p>
```

Comment:
Could you somehow check here if it's daily or weekly, and display accordingly? If it's too much, we can do it in a follow-up PR. This is great!

Reply:
Okay so I am trying to do this, but I think that it is probably hard coded somewhere, and a variable for checking which type of digest is not being sent to the actions:

Like here, in the mailer code:
https://github.com/publiclab/plots2/blob/d3b122efaa42a9d6b06a70f53626334ad2cf085b/app/mailers/subscription_mailer.rb#L58

Here in the send_digest_email function, top picks are being selected for a week only:

https://github.com/publiclab/plots2/blob/d3b122efaa42a9d6b06a70f53626334ad2cf085b/app/models/user.rb#L325-L330

I think here, when send_digest_email function is being called, the frequency parameter should be passed somehow, and received everywhere, then we can correct the code in the send_digest.html.erb:

https://github.com/publiclab/plots2/blob/d3b122efaa42a9d6b06a70f53626334ad2cf085b/app/jobs/digest_mail_job.rb#L4-L14

Please correct me if I am wrong anywhere, then we can explore this better.

Reply:
@jywarren any updates here?

Reply:
You're totally right. Let's leave it for now in this PR and if you could copy your notes into a new issue (linking from #4732 ) we can address this with some refactoring! Great sleuthing, @IshaGupta18 !!!

Reply:
Absolutely @jywarren! Thanks a lot!